manual/sourcetree-disableautoupdate/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,4 @@
+﻿$ErrorActionPreference = 'Stop'
+
+# Disable auto update
+New-Item 'HKCU:\Software\Caphyon\Advanced Updater\{138B87CA-4905-43FE-BB63-184FFB0CE1EC}\Settings' -Force | New-ItemProperty -Name AutoUpdatePolicy -Value 0 -Force
```

Comment:
as per [2.2 Formatting](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting), you're missing an empty line at the end of the file.